### ISSUE TITLE
Change Cupcake result, fix unicorn to be truly rare

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#1953]
- [ ] The unicorn ascii art shown on zero warns/fails was added because this was rare. But the fun gamification reward of no FAILs should be common! :) And it will be, once #1939 #1940 and #1941 are fixed. So lets add a small cupcake ascii art to terminal output when no FAILs occur.
- [ ] If you use the `-c` arg to only run one check, you'll commonly see all checks pass and get a unicorn, unfairly ;)

As a reference, this is our current unicorn ascii-art:
![screenshot at 2018-07-09 17 52 51](https://user-images.githubusercontent.com/213676/42475428-25c0d666-83ba-11e8-9efb-d63bd1daf2a6.png)

**Comment by felipesanches** [comment]
haha! That's a lovely proposal :-)

I wonder where could we find some free (as in freedom) cupcake ascii-art though...

**Comment by davelab6** [comment]
Use a image2asciiart filter?

**Comment by chrissimpkins** [comment]
```
                    \
                     @                    
                 #;;;;;;;#                
                ;;;+;;;;;;;               
               ;;;;;(;;;O;;;              
              ;;+@;;;;;;;;;;+             
            ;;;@+;;;;;;;;+;;;;+          
          +;;;;;;;;;;;;;;);;;;;;;@        
       #;;;;(;;;;;;+;;;;;;;;;';;;;#       
      ;;;;;;;;;;;;;;;;;;;;;;;;@;;;;       
      @;;;;;;;;;O;;;;;;;;;+;;;;;(;;;;#         
   +;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;@      
 @;;;;;;;O;;;;;;;;;;O;;;;;;);;;+;;;;O;;;;'
 ;;@;;;;;;;;;);;;;;;;;;;;;;;;;;;;;;;;;;;;'
 ;@;';;;;;+;;;;;;;;;;;;;;;@;;;;;;;;#';;;;#
 ;;#;;;;;;;;;;;(;;;;;+;;;;;;;;;;(;;;;;;;; 
 ;;;;;;;;O;;;;;;;;;;;;;;;;;;);;;;;;+;;;;@ 
  ;;;;;);;;;;;;;;;;O;;;;;;;;;;;+;;;;;;;   
   ;;;;;;;;;;;;;;;;;;;;+#+;;;;;;;;;;'@#   
   #,'@@@',..+@++#@+......;.;@#@@@;...#    
   $...$.......$..........$......$...$    
    ...'.......;..........@......@...;    
    @...;......@..........+......'...,    
    '...#......#..........;.....;....'        
     '...@......;........#.....,....#      
     @...;......+........@.....#...@      
      @...,.....@.......;......@...,      
      #...;.....#.......,......;...#      
      ;...@.....;.......;.....,....       
      @...#.....%.......+.....;....       
       ;..;.....;.......@.....'...@       
         @+.....,.......#......@@          
               @@#';,;;+@@@            
```

CC0

**Comment by davelab6** [comment]
Haha nice :) we can't accept it under cc0, will Apache be ok?

**Comment by chrissimpkins** [comment]
Absolutely :)  Updated

**Comment by felipesanches** [comment]
I still find the unicorn more aesthetically pleasing. I'll leave this open for some more time until we hopefully find a nicer looking image to replace it.

As this is the lowest-priority of all issues, I'll surely move this to a future release milestone.

**Comment by felipesanches** [comment]
@graphicore you gave us the unicorn to begin with, so maybe you have some ideas here as well? :-)

**Comment by felipesanches** [comment]
I made an experimental commit with @chrissimpkins artwork. I am not convinced yet. But I'll leave it here as a reference of his proposal.

![screenshot at 2018-07-09 19 17 04](https://user-images.githubusercontent.com/213676/42478906-ff7900b2-83c5-11e8-925d-b96ce6f89a86.png)

**Comment by felipesanches** [comment]
I contacted a friend today and asked him to design something based on a brief of what we need. I cannot guarantee that his work will be good but I'd like to give it a try and see what does he come up with.

**Comment by felipesanches** [comment]
To make clear what's my criticism of Chris' cupcake:

* It is too big if compared to the space available in a single screen on the terminal (without needing to scroll). The previous artwork did fit nicely in the available space. But I had to tweak the font size in my virtual terminal in order to take the screenshot above with the proposed cupcake.
* It is filled with characters and dithering effects, which clearly indicates it was likely automatically generated by passing an image into a special purpose ascii-art filter. The original unicorn artwork on the other hand was carefully designed by an ascii-artist, carefully choosing which characters would best convey the artwork outlines. I'd like to keep that style as our users are all type designers who value such extreme attention to details.
* The cherry on top is disturbingly small LOL

So, let's wait until I get some response from my friend and I'll let you all know what we've got. I've used these insights that were gathered by observing @chrissimpkins proposal while preparing a brief of what I'd like to see. So I consider that Chis' proposal is indeed useful and welcome :-) Thanks!

**Comment by chrissimpkins** [comment]
No worries! Use it if helpful, pitch it if not.

**Comment by chrissimpkins** [comment]
and if you do happen to use it, no attribution is necessary.  It was auto generated with web based software and then touched up with lines on the cup, sprinkles, and a cherry on top :) 

please don't be concerned about use of this.  if you don't like it, it really is not a big deal.  I couldn't find anything with an open license when I searched.   cupcakes do not appear to be in demand in the ascii art world :)

**Comment by felipesanches** [comment]
> cupcakes do not appear to be in demand in the ascii art world :)

Now they are! LOL

**Comment by felipesanches** [comment]
I commissioned this artwork from a friend, a brazilian type designer called Tony de Marco:

![screenshot at 2018-07-26 19 53 51](https://user-images.githubusercontent.com/213676/43292914-2310d694-9127-11e8-88a4-97ae8a2da2fb.png)

**Comment by felipesanches** [comment]
Tony's cupcake ascii-art is made available under Apache, v2.0
